lib/invoiced/util.rb

```diff
@@ -17,7 +17,23 @@ def build_objects(_class, objects)
             end
 
             def convert_to_object(_class, values)
-                object = _class.class.new(_class.client, values[:id], values)
+                class_override = nil
```

Comment:
It might be cleaner to assign this to `_class.class` by default

Reply:
ah, agree

Reply:
done